tests/cypress/integration/actions_tasks_objects/case_66_rename_label_raw_editor.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Rename a label via raw editor.', () => {
+    const caseId = '66';
+    const labelName = `Case ${caseId}`;
+    const taskName = `New annotation task for ${labelName}`;
+    const attrName = `Attr for ${labelName}`;
+    const textDefaultValue = 'Some default value for type Text';
+    const imagesCount = 1;
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    const newlabelName = `Changed case ${caseId}`;
+    const newlabelColor = '#C14330';
+    let rawLabelViewerDefault = '';
+
+    function testChangingRawLabelsViewerText(rawLableViewer) {
```

Comment:
typos, better to rename as ``rawLabelsInput``

Reply:
Renamed.